compliance/component.yaml

```diff
@@ -0,0 +1,237 @@
+schema_version: 3.0.0
+name: FedRAMP Dashboard
+documentation_complete: false
+references:
+- name: Amazon Web Services' S3 Authorized URLs
+  path: http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-query-string-auth.html
+  type: URL
+- name: New Relic Application Monitoring
```

Comment:
Technically we are just using New Relic's Browser monitoring because it is a static site